activation/activation.go

```diff
@@ -301,6 +317,39 @@ func (b *Builder) generateProof() error {
 	return nil
 }
 
+func (b *Builder) waitForFirstATX(ctx context.Context) {
+	if b.layerClock.GetCurrentLayer().GetEpoch() == 0 { // genesis miner
+		return
+	}
+	if _, err := b.cdb.GetPrevAtx(b.nodeID); err == nil { // not a new miner
+		return
+	}
+
+	// first atx for this node
+	currentLayer := b.layerClock.GetCurrentLayer()
+	waitTill := (currentLayer.GetEpoch() + 1).FirstLayer()
+	b.log.WithContext(ctx).With().Info("building very first atx",
+		log.Stringer("current_epoch", currentLayer.GetEpoch()),
+		log.Stringer("wait_till", waitTill),
+		log.Stringer("wait_till_epoch", waitTill.GetEpoch()))
+	select {
+	case <-ctx.Done():
+		return
+	case <-b.layerClock.AwaitLayer(waitTill):
+		// this estimate work if the majority of the nodes use poet servers that are configured the same way.
+		// TODO: do better when nodes use different poet services
+		timer := time.NewTimer(time.Second * 35)
```

Comment:
comment doesn't make sense.
are you waiting for next epoch or 35 seconds after the epoch? what is special about those 35 seconds?